zh-tw/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="zh-Hans">
+<head>
+  <title>不要為了提問而提問</title>
+  <meta property="og:title" content="不要為了提問而提問，請直接提問" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com/zh-cn" />
+  <meta property="og:locale" content="zh_CN" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Noto+Sans+SC">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <style>
+    body {
+      font-family: "Noto Sans SC" !important;
+    }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>不要為了提問而提問</h1>
+    <p>
+      在我窺屏的一些聊天群組中，時不時會有人問出類似於這樣的問題：
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        有沒有精通 Java 的？
+      </p>
+    </blockquote>
+    <p>
+      啊，無論你怎麼解釋，這真的不是個好問題。提出問題的人
+      <em>真正</em>想問的其實是：
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        有沒有願意幫我解決問題的 Java 高手？
+        雖然我也不知道問題出在哪裡，也許可能不是 Java 的問題。
+        或者有沒有不精通 Java 但願意幫我解決一下這個問題的群友？
+      </p>
+    </blockquote>
+    <p>
+      那些確實精通這方面的人會有一萬種理由不接受你的問題。
+      對於這個問題，你對他們要求的比你預想中的要多很多。
+    </p>
+    <p>
+      這意味著你要讓回答你問題的人承擔一定的責任。
+      這個問題像是在挑戰他們對這方面知識和能力的信心，同時也無意中將他們拒之門外。
+      我經常回答一些和我從未使用過的語言或程序庫有關的問題，因為那些問題的答案（對於我們程序員來說）往往都是常識。
+    </p>
+    <p>
+      另外還有一種問題：
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        我有一個關於 Java 的問題，但我太懶了，不想用語言解釋出來，除非頻道 / 群組裡有人可以回答我。
+      </p>
+    </blockquote>
+    <p>
+      天哪，這也太懶了吧。如果連你自己都不願意幫助自己解決問題，那更何況別人呢？
+    </p>
+    <p>
+      這種問題是很好避免的。不要為了提問而提問，直接表達你的問題即可。
+      那些在頻道 / 群組裡窺屏的人只是時不時看看這裡有什麼事情發生，並不太可能會去回答那些 “問裡有問” 的問題。
+      往往那些帶有實際描述的問題才會激起他們的興趣，並讓他們回答或協助解決。
+    </p>
+    <p>
+      因此，總的來說，不要問
+      <em>“有沒有 Java 高手？”</em>這類問題，
+      而是去問諸如
+      <em>“如何使用 Java （以及其他相關的描述）去處理某個問題”</em>
+      的問題。
+    </p>
+    <p>
+      其他類似的問題：XY 問題（<a href="http://xyproblem.info/">英文</a> <a href="http://www.aqee.net/post/xy-problem.html ">中文</a>）、<a href="https://www.nohello.com/">提問切勿打招呼（英文）</a>。 <br/>
+      延伸閱讀：<a href="https://stackoverflow.com/help/how-to-ask">Stack Overflow 如何問一個好問題？ （英文）</a>。 <br/>
+      或者如果你有時間的話，可以讀一讀<a href="https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/master/README-zh_CN.md" >提問的智慧</a>。
+    </p>
+  </main>
+  <footer>
+    本頁面多數內容來自 <a href="http://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+    -
+    由 <a href="https://github.com/cyanoxygen">Cyanoxygen</a> 翻譯
```

Comment:
I guess this is based on the zh-cn translation, but if you want you can update the credits so it mentions both you and the original translation.

You can also add it to the list in the readme